crypto/dilithium/p_dilithium_test.cc

```diff
@@ -665,6 +668,70 @@ TEST(Dilithium3Test, SIGOperations) {
   md_ctx.Reset();
 }
 
+static void RunKat(FileTest *t)
+{
+  std::string count, mlen, smlen;
+  std::vector<uint8_t> seed, msg, pk, sk, sm;
+
+  ASSERT_TRUE(t->GetAttribute(&count, "count"));
+  ASSERT_TRUE(t->GetBytes(&seed, "seed"));
+  ASSERT_TRUE(t->GetAttribute(&mlen, "mlen"));
+  ASSERT_TRUE(t->GetBytes(&msg, "msg"));
+  ASSERT_TRUE(t->GetBytes(&pk, "pk"));
+  ASSERT_TRUE(t->GetBytes(&sk, "sk"));
+  ASSERT_TRUE(t->GetAttribute(&smlen, "smlen"));
+  ASSERT_TRUE(t->GetBytes(&sm, "sm"));
+
+  //the KAT files generated by the dilithium team use the optional APIs that
```

Comment:
What's happening with the missing spaces after `//`? (:

Reply:
i guess that's jake's comment style? i can add some spaces in there.